autonick.lua

```diff
@@ -0,0 +1,56 @@
+-- automatically nickname dwarves
+--[====[
+
+autonick
+========
+Automatically nickname dwarves according to ``dfhack-config/autonick.txt``.
```

Comment:
This might be a bit clearer:

Gives dwarves unique nicknames chosen randomly from ``dfhack-config/autonick.txt``.